src/SnackbarProvider.js

```diff
@@ -16,13 +24,7 @@ class SnackbarProvider extends Component {
     get offsets() {
         const { snacks } = this.state;
         return snacks.map((item, i) => {
-            let index = i;
-            let offset = 20;
-            while (snacks[index - 1]) {
-                offset += snacks[index - 1].height + 16;
-                index -= 1;
-            }
-            return offset;
+            return 20;
```

Comment:
This function seems to be useless now.
Instead of calling this.offsets[index], a constant is enough.